Recrutify/Recrutify.Host/Controllers/EmailController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Recrutify.Services.Services.Abstract;
+
+namespace Recrutify.Host.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
can remove it, save just locally to use in next task, but now it's not necessary